docs/release-notes/1.9.0.asciidoc

```diff
@@ -0,0 +1,78 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-1.9.0]]
+== {n} version 1.9.0
+
+[[breaking-1.9.0]]
+[float]
+=== Breaking changes
+
+* Inherit resource requirements in init containers {pull}4963[#4963]
+
+All init containers will inherit resource requirements from the main containers. As this changes the `podTemplate` used, upgrading ECK will cause rolling update to all ECK-managed Pods.
+
+* Move to env vars based Fleet mode configuration {pull}4889[#4889]
+
+ECK operator will configure Elastic Agent in Fleet mode using link:https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[environmental variables] instead of configuration file. This makes overriding default settings much easier, but requires an additional step before the operator upgrade if such an override was configured using any other method.
```

Comment:
```suggestion
ECK operator will configure Elastic Agent in Fleet mode using link:https://www.elastic.co/guide/en/fleet/current/agent-environment-variables.html[environmental variables] instead of a configuration file. This method allows you to easily override the default settings. If the override is configured with any other method, an additional step is required before the operator upgrade.
```